### PR TITLE
Translate train-optimize-sd-intel.md to zh

### PR DESCRIPTION
**Pull request by Vermillion-de**
There are some little questions left to be , maybe edit further.

### CODE REVIEW

File: zh/train-optimize-sd-intel.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+基于隐空间的扩散模型（Latent Diffusion Model），是解决文本到图片生成问题上的颠覆者。Stable Diffusion 是最著名的一例，广泛应用在商业和工业。Stable Diffusion 的想法简单且有效：从噪声向量开始，多次去噪，以使之在隐空间里逼近图片的表示。
+
+但是，这样的方法不可避免地增加了推理时长，使客户端的体验大打折扣。众所周知，一个好的 GPU 总能有帮助，确实如此，但其损耗大大增加了。就推理而言，在 2023 年上半年（H1'23），一个好 CPU 实例（[r6i.2xlarge](https://aws.amazon.com/ec2/instance-types/r6g/) ，8 vCPUs ，64 GB 内存）价格是 0.504 $/h，同时，类似地，一个好 GPU 实例（[g4dn.2xlarge](https://aws.amazon.com/ec2/instance-types/g4/)，NVIDIA T4，16 GB 内存）价格是 0.75 $/h ，是前者的近 1.5 倍。
+
+
+这就使图像生成的服务变得昂贵，无论持有者还是用户。该问题在面向用户端部署就更突出了：可能没有 GPU 能用！这让 Stable Diffusion 的部署变成了棘手的问题。
+
+在过去五年中，[OpenVINO](https://docs.openvino.ai/) 集成了许多高性能推理的特性。 其一开始为计算机视觉模型设计，现今仍在许多模型的推理性能上取得最佳表现，包括 [Stable Diffusion](https://huggingface.co/blog/stable-diffusion-inference-intel)。然而，对资源有限型的应用，优化 Stable Diffusion 远不止运行时的。这也是 OpenVINO [NNCF](https://github.com/openvinotoolkit/nncf)(Neural Network Compression Framework) 发挥作用的地方。
+
+在本博客中，我们将理清优化 Stable Diffusion 模型的问题，并提出对资源有限的硬件（比如 CPU）减负的流程。特别地，和 PyTorch 相比，我们速度提高了 5.1 倍，内存减少了 4 倍。 <!--考证如何翻译-->
```
Comment (zhongdongy) [inline zh/train-optimize-sd-intel.md:26]:
```plain
<!--考证如何翻译-->
```
In the submitted version, please do not include such discussions. These should be processed during translation, perhaps in the group chats?

File: zh/train-optimize-sd-intel.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+# 对 Stable Diffusion 基于 NNCF 和 🤗 Optimum 的面向 Intel CPU 优化 
+
+基于隐空间的扩散模型（Latent Diffusion Model），是解决文本到图片生成问题上的颠覆者。Stable Diffusion 是最著名的一例，广泛应用在商业和工业。Stable Diffusion 的想法简单且有效：从噪声向量开始，多次去噪，以使之在隐空间里逼近图片的表示。
+
+但是，这样的方法不可避免地增加了推理时长，使客户端的体验大打折扣。众所周知，一个好的 GPU 总能有帮助，确实如此，但其损耗大大增加了。就推理而言，在 2023 年上半年（H1'23），一个好 CPU 实例（[r6i.2xlarge](https://aws.amazon.com/ec2/instance-types/r6g/) ，8 vCPUs ，64 GB 内存）价格是 0.504 $/h，同时，类似地，一个好 GPU 实例（[g4dn.2xlarge](https://aws.amazon.com/ec2/instance-types/g4/)，NVIDIA T4，16 GB 内存）价格是 0.75 $/h ，是前者的近 1.5 倍。
+
+
+这就使图像生成的服务变得昂贵，无论持有者还是用户。该问题在面向用户端部署就更突出了：可能没有 GPU 能用！这让 Stable Diffusion 的部署变成了棘手的问题。
+
+在过去五年中，[OpenVINO](https://docs.openvino.ai/) 集成了许多高性能推理的特性。 其一开始为计算机视觉模型设计，现今仍在许多模型的推理性能上取得最佳表现，包括 [Stable Diffusion](https://huggingface.co/blog/stable-diffusion-inference-intel)。然而，对资源有限型的应用，优化 Stable Diffusion 远不止运行时的。这也是 OpenVINO [NNCF](https://github.com/openvinotoolkit/nncf)(Neural Network Compression Framework) 发挥作用的地方。
```
Comment (zhongdongy) [inline zh/train-optimize-sd-intel.md:24]: See last comment on HF blog URL manipulation.

**Review by zhongdongy — CHANGES_REQUESTED** [review]
Thanks Vermillion, I've added some comments, could you please check and update?

**Comment by zhongdongy** [comment]
@Vermillion-de Hi, just FYI, you will also need to insert an entry to `zh/_blog.yml` file, just like other ones. You can find the original one from [HF official repo](https://github.com/huggingface/blog/blob/main/zh/_blog.yml). Please also note the entries in `zh/_blog.yml` are sorted by `date` field, appreciate that if you can also follow this convention. 🤗

File: zh/stacklamma.md
[the remark this answers is not in the document]
Comment (Vermillion-de) [reply zh/stacklamma.md:1]: > Thanks Vermillion, I've added some comments, could you please check and update?

Thank you! Will check that out!

File: zh/train-optimize-sd-intel.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+authors:
+- user: AlexKoff88
+  guest: true
+- user: MrOpenVINO
+  guest: true
+- user: helenai
+  guest: true
+- user: sayakpaul
+- user: echarlaix
+---
```
[the remark this answers is not in the document]
Comment (Vermillion-de) [reply zh/train-optimize-sd-intel.md:13]: Wow, that is cool!

File: zh/train-optimize-sd-intel.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+
+```
+
+在 Hugging Face [Optimum Intel](https://huggingface.co/docs/optimum/main/en/intel/index) 库中你可以找到训练和量化[代码](https://github.com/huggingface/optimum-intel/tree/main/examples/openvino/stable-diffusion)。比较优化过的和原模型的 notebook 代码在[这里](https://github.com/huggingface/optimum-intel/blob/main/notebooks/openvino/stable_diffusion_optimization.ipynb)。你可以在 Hugging Face Hub 上找到 [OpenVINO](https://huggingface.co/OpenVINO) 下的[许多模型](https://huggingface.co/models?library=openvino&sort=downloads)。另外，我们在 Hugging Face Spaces 上建了一个 [demo](https://huggingface.co/spaces/AlexKoff88/stable_diffusion)，以运行 [带第三代 Intel Xeon Scalable 的 r6id.2xlarge 实例]。
+
+## 一般的 Stable Diffusion 模型呢？
+
+正如我们在宝可梦图像生成任务中展现的一样，仅用小量的训练资源，对 Stable Diffusion 管线实现高层次的优化是可能的。同时，众所周知，训练一般的 Stable Diffusion 模型是一个[昂贵的任务](https://www.mosaicml.com/blog/training-stable-diffusion-from-scratch-part-2)。但是，有充足的资金和硬件资源，用上述方法优化一般的模型生成高分辨率的模型是可能的。我们唯一的警告是关于 Token Merging 方法，其会减弱模型容忍性。这里衡量标准是，训练数据越复杂，优化模型时的融合比就该越小。
+
+如果你乐于读本博客，那你可能对另外一篇[博客](https://huggingface.co/blog/stable-diffusion-inference-intel)感兴趣，它讨论了在第四代 Intel Xeon CPU 上其他互补的 Stable Diffusion 模型优化方法。
```
[the remark this answers is not in the document]
Comment (Vermillion-de) [reply zh/train-optimize-sd-intel.md:107]: Oh yeah, indeed!

File: zh/train-optimize-sd-intel.md (hunk 1 above)
Comment (Vermillion-de) [reply zh/train-optimize-sd-intel.md:26]: Yeap, will do that soon!

**Comment by Vermillion-de** [comment]
> 

Got it!

**Comment by zhongdongy** [comment]
@Vermillion-de Any updates to this PR? 🤗

**Pull request merged**